registration/start.js

```diff
@@ -0,0 +1,47 @@
+document.querySelector(`#form_id`).addEventListener(`submit`, function (e) {
+    e.preventDefault();
+    if(signIn()) {
+        let str = ["spider","css-form","spiderNetBotR","displayed"];
+        for (let i in str) {
+            document.querySelector(`.${str[i]}`).classList.add("hide");
+        }
+        document.querySelector('body').classList.add("strawberry");
+    }
+    return false;
+});
+function signIn() {
+
+    let names = document.querySelectorAll('#form_id input');
+    const objForm = [];
+
+    for (let k = 0; k < names.length - 1; k++) {
+        objForm[k] = names[k].value;
+    }
+
+    for (let x in objForm) {
+
+        switch (+x) {
+            case 0:
+                if (!/[0-9a-zа-я_A-ZА-Я]+@[0-9a-zа-я_A-ZА-Я^.]+\.[a-zа-яА-ЯA-Z]{2,4}/i.test(objForm[x])) {
+                   return  warning("Please type a correct email!");
+                }
+                break;
+            case 1:
+                if (!/^([а-яА-ЯA-Za-z0-9]{6,})+$/.test(objForm[x])) {
+                   return  warning("password must include more then 6 simbols!");
+                }
+                break;
+        }
+    }
```

Comment:
General observation - use human-readable names. For ex. - wtf is `case 0` - it is understandable for you, but not for another developer. Use `node.name` instead for switch-case recognization.
Again, why we have two loops for this simple check operations?